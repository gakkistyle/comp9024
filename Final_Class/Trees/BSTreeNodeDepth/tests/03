1 8 3 2 6 5 4 7 9
0
1
2
3
4
5
6
7
8
9
10
