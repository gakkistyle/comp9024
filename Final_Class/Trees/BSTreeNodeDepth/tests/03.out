Enter the preorder traversal of the BST: Tree:

  1
   \
    8
   / \
  3   9
 / \
2   6
   / \
  5   7
 /
4

Enter key: For key = 0, BSTreeNodeDepth returned -1
Enter key: For key = 1, BSTreeNodeDepth returned 0
Enter key: For key = 2, BSTreeNodeDepth returned 3
Enter key: For key = 3, BSTreeNodeDepth returned 2
Enter key: For key = 4, BSTreeNodeDepth returned 5
Enter key: For key = 5, BSTreeNodeDepth returned 4
Enter key: For key = 6, BSTreeNodeDepth returned 3
Enter key: For key = 7, BSTreeNodeDepth returned 4
Enter key: For key = 8, BSTreeNodeDepth returned 1
Enter key: For key = 9, BSTreeNodeDepth returned 2
Enter key: For key = 10, BSTreeNodeDepth returned -1
Enter key: 